.env.example

```diff
@@ -1,3 +1,9 @@
 ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
 ROPSTEN_URL=https://eth-ropsten.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 PRIVATE_KEY=0xabc123abc123abc123abc123abc123abc123abc123abc123abc123abc123abc1
+
+# Optional for a Goerli Deployment
+NODE_5=https://goerli.infura.com/xxx
+
+# Optional for customizing Infura endpoint API
+INFURA_API_KEY=
```

Comment:
Sorry a little confused about these values. Is only one of these required? Or are both required? If we have the INFURA_API_KEY, do we still need each url specified separately?